unfoldingWord/translationCore
# Add Close Button To Popovers

#### This pull request addresses:
Bring explicit closing functionality to popovers

#### How to test this pull request:
Check the popovers in the app and ensure clicking the x at the top right closes them


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2825)
